spec/models/vmdb_database_connection_spec.rb

```diff
@@ -1,16 +1,15 @@
 require "concurrent/atomic/event"
 
-describe VmdbDatabaseConnection do
-  before do
-    @db = FactoryGirl.create(:vmdb_database)
-  end
+RSpec.configure do |c|
+  c.use_transactional_examples = false
+end
 
+describe VmdbDatabaseConnection do
```

Comment:
If you want to avoid the `RSpec.configure` stuff, you might be able to do:

```ruby
describe VmdbDatabaseConnection, :use_fixtures => false do
```

https://github.com/rspec/rspec-rails/blob/a89eee761659d197e5c9df93b2f339f919795237/lib/rspec/rails/configuration.rb#L69

Would need to test this though.

Reply:
So, I think I am currently not able to reproduce this issue.  I first removed all of the `RSpec.configure` blocks you added to confirm that the test would fail as you described, but that didn't seem to cause anything to fail when running:

```console
$ bundle exec rspec spec/models/vmdb_database_connection_spec.rb
```

Not sure if I need to do something different to get this to fail.

Reply:
Rails 5.1?  This test file hangs with a deadlock for me and Travis (our diligent coworker who runs any builds we ask for)

Reply:
Derp, I was using this branch, which doesn't have Rails 5.1 on it.  Hold plz...

Reply:
@jrafanie Okay, took a bit, but the "simplest" way to do this for just this spec is:

```ruby
describe VmdbDatabaseConnection do
  self.use_transactional_tests = false
  # ...
```

Should be localized to just this spec as well.  I can give you the ":book: answer" if you want, but will save that for now.

Reply:
Thanks, I thought I tried that but I'll give it another go.

Reply:
Cool, looks good.  Resolving.